app/communication/index.ts

```diff
@@ -0,0 +1,23 @@
+import { isEmpty } from "lodash";
+import { json } from "remix";
+import { IError } from "~/types";
+import { getTransformedError } from "~/helpers";
+
+export const apiCall: (url: string, requestHeaders?: HeadersInit,  method?: string, requestBody?: any) => Promise<Response> = async (url: string, requestHeaders?: HeadersInit, method: string = "GET", requestBody?: any): Promise<Response> =>
```

Comment:
it could be good if we have specific:
- post
- get
- etc
so when we use it is easy to identify what it does